src/main.cpp

```diff
@@ -17,23 +17,23 @@ int main(int argc, char *argv[])
     KiwixApp a(argc, argv);
     a.setApplicationVersion(version);
     QCommandLineParser parser;
-    QString zimfile;
     parser.setApplicationDescription(QStringLiteral("The Kiwix Desktop is a viewer/manager of ZIM files for GNU/Linux and Microsoft Windows OSes."));
     parser.addHelpOption();
     parser.addVersionOption();
     parser.addPositionalArgument("zimfile", "The zim file");
     parser.process(a);
     auto positionalArguments = parser.positionalArguments();
-    if (positionalArguments.size() >= 1) {
-        zimfile = parser.positionalArguments().at(0);
-    }
     if (a.isRunning()) {
-        a.sendMessage(zimfile);
+        for(QString zimfile : positionalArguments) {
```

Comment:
Maybe just put a space after the `for`

Reply:
Doing. Somehow I always seem to do this mistake :facepalm: